$NetBSD: patch-Makefile,v 1.4 2023/02/04 09:25:50 kim Exp $

- Don't make/install PDF.
- Add XMLTO_FLAGS.
- Do not install tigrc, this is handled via CONF_FILES by pkgsrc.

--- Makefile.orig	2022-08-25 17:27:20.000000000 +0000
+++ Makefile
@@ -51,7 +51,7 @@
 TXTDOC	= doc/tig.1.adoc doc/tigrc.5.adoc doc/manual.adoc NEWS.adoc README.adoc INSTALL.adoc test/API.adoc
 MANDOC	= doc/tig.1 doc/tigrc.5 doc/tigmanual.7
 HTMLDOC = doc/tig.1.html doc/tigrc.5.html doc/manual.html README.html INSTALL.html NEWS.html
-ALLDOC	= $(MANDOC) $(HTMLDOC) doc/manual.html-chunked doc/manual.pdf
+ALLDOC	= $(MANDOC) $(HTMLDOC) doc/manual.html-chunked
 
 # Never include the release number in the tarname for tagged
 # versions.
@@ -70,6 +70,7 @@
 ASCIIDOC ?= asciidoc
 ASCIIDOC_FLAGS = -aversion=$(VERSION) -asysconfdir=$(sysconfdir) -f doc/asciidoc.conf
 XMLTO ?= xmlto
+XMLTO_FLAGS =
 DOCBOOK2PDF ?= docbook2pdf
 
 LCOV ?= lcov
@@ -134,7 +135,6 @@
 
 install: all
 	$(QUIET_INSTALL)tools/install.sh bin $(EXE) "$(DESTDIR)$(bindir)"
-	$(QUIET_INSTALL)tools/install.sh data tigrc "$(DESTDIR)$(sysconfdir)"
 
 install-doc-man: doc-man
 	$(Q)$(foreach doc, $(filter %.1, $(MANDOC)), \
@@ -440,10 +440,10 @@
 	$(QUIET_ASCIIDOC)$(ASCIIDOC) $(ASCIIDOC_FLAGS) -b docbook -d article $<
 
 % : %.xml
-	$(QUIET_XMLTO)$(XMLTO) man -o doc $<
+	$(QUIET_XMLTO)$(XMLTO) $(XMLTO_FLAGS) man -o doc $<
 
 %.html-chunked : %.xml
-	$(QUIET_XMLTO)$(XMLTO) html -o $@ $<
+	$(QUIET_XMLTO)$(XMLTO) $(XMLTO_FLAGS) html -o $@ $<
 
 %.pdf : %.xml
 	$(QUIET_DB2PDF)$(DOCBOOK2PDF) -o doc $<
